Makefile

```diff
@@ -1,6 +1,6 @@
 #_*_MakeFile_*_
 CC = gcc 
-_CFLAGS = -Os -s -g -std=gnu99
+_CFLAGS = -O2 -s -g -std=gnu99
```

Comment:
This one-character change however, is very consequential. This should be its own commit, with a longer detailed message explaining why this change is mad. Consider someone else trying to debug an issue, and running bisect -- this may be important information to someone in the future using an old GCC wondering why they are seeing the issue again, etc.